crypto/src/main/java/net/corda/v5/crypto/KeyUtils.java

```diff
@@ -0,0 +1,48 @@
+package net.corda.v5.crypto;
+
+import java.security.PublicKey;
+import java.util.Collections;
+import java.util.Set;
+
+public class KeyUtils {
+
+    /**
+     * Constant specifying the maximum number of key lookup by id items.
+     */
+    public static int KEY_LOOKUP_INPUT_ITEMS_LIMIT = 20;
+
+
+    public static boolean containsAny(PublicKey key, Iterable<PublicKey> otherKeys ) {
```

Comment:
This is probably not a good name now that it isn't an extension function anymore, especially given the order of the arguments. With the arguments swapped the other way around, maybe it would work?

As an aside, I thought we were moving the interfaces/classes first, then extension methods in a second PR?